app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -115,7 +115,7 @@ def build_form
     end
 
     def actor
-      @actor ||= CurationConcerns::CurationConcern.actor(curation_concern, current_user, attributes_for_actor)
+      @actor ||= CurationConcerns::CurationConcern.actor(curation_concern, current_user)
```

Comment:
I like this disentanglement. The `CurationConcerns::CurationConcern.actor` method is a query method (building an instance). The eventual `#update` method is the state changing thing based on attributes. It is the command.